installer.sh

```diff
@@ -15,9 +15,26 @@ echo "to install all necessary things. It will prompt you to enter your"
 echo "password when required."
 echo
 echo
-
-if [ "$(lsb_release -i -s)" != "Ubuntu" ]; then
-	echo "ERROR: You are running the installer on a not support distribution."
+#Array of supported distributions. Just add any working distributions here
+#Careful: LinuxMint will do systemd installation because the version number won't match Ubuntu's. If you add a distribution without systemd which does not follow the same numbering as Ubuntu, it will try systemd installation instead of Upstart
+
+supported_dist=("Ubuntu" "LinuxMint")
+DISTRIB_ID="$(lsb_release -i -s)"
+
+function contains() {
+    local n=$#
+    local value=${!n}
+    for ((i=1;i < $#;i++)) {
+        if [ "${!i}" == "${value}" ]; then
+		echo "y"
+            	return 0
+        fi
+    }
+    return 1
+}
+
+if [ "$(contains "${supported_dist[@]}" "$DISTRIB_ID")" != "y" ]; then
+	echo "ERROR: You are running the installer on an unsupported distribution."
 	echo "       At the moment we only support Ubuntu."
```

Comment:
Can you print out a list of support distributions here? As "only Ubuntu" isn't true anymore now :-)